### ISSUE TITLE
Documentation for the nested type `Construct_initial_points` of `MeshDomain_3` is wrong

### ISSUE THREAD

**Issue by burnpanck** [CGAL/cgal#6098]
## Issue Details

The documentation states that the following two signatures are required for that type:
```c++
template<typename OutputIterator>
OutputIterator operator()(OutputIterator pts);

template<typename OutputIterator>
OutputIterator operator()(int n, OutputIterator pts);
```

However, it seems that the package actually expects swapped argument order for the two argument signature.
That is also the more convenient signature, because it allows both signatures to be implemented with a single function, 
using a default value for n.

## Source Code

The example from my previous issue #6097 can easily be adapted to test this observation.

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits):
* Compiler:
* Release or debug mode:
* Specific flags used (if any):
* CGAL version: 5.2.1
* Boost version:
* Other libraries versions if used (Eigen, TBB, etc.):

**Comment by sloriot** [comment]
@janetournois do we still what that doc fix for 5.2.4 or shall we changed the milestone to 5.5 (or internal bug fix release)?

**Comment by janetournois** [comment]
I will fix it in PR #5693, which already fixes several similar errors
